scripts/extensions/auto-tagging-widget/src/ImageTaggingComponent/ImageTaggingComponent.tsx

```diff
@@ -0,0 +1,278 @@
+import * as React from "react";
+import { ISuperdesk } from "superdesk-api";
+import { ToggleBoxNext } from "superdesk-ui-framework";
+
+interface ITag {
```

Comment:
Is this the same entity as `ITagBase`? If so, let's use the existing interface

Reply:
It's not same as ITagBase, it's in another format and is used in the request payload. This has been refactored to ITagInput. Thanks for the feedback!